source/cl/source/kernel.cpp

```diff
@@ -1568,6 +1568,11 @@ CL_API_ENTRY cl_int CL_API_CALL cl::EnqueueNDRangeKernel(
   cl_event return_event = nullptr;
   cl_int error = 0;
 #ifdef OCL_EXTENSION_cl_intel_unified_shared_memory
+  // we need to lock the context for the remainder of the function as we need to
+  // ensure any blocking operations such clMemBlockingFreeINTEL are entirely in
+  // sync as createBlockingEventForKernel adds to usm lists assuming that they
```

Comment:
```suggestion
  // sync as createBlockingEventForKernel adds to USM lists assuming that they
```More consistent with `USM` used in other comments (the one below this)